src/overlays/actors/ovl_En_Fr/z_en_fr.h

```diff
@@ -5,6 +5,9 @@
 
 struct EnFr;
 
+#define ENFR_GET_SWITCHFLAG(thisx) (((thisx)->params & 0x7F))
+#define ENFR_GET_TARGETMODE(thisx) (((thisx)->params >> 7 & 0xF))
```

Comment:
```suggestion
#define ENFR_GET_SWITCHFLAG(thisx) ((thisx)->params & 0x7F)
#define ENFR_GET_TARGETMODE(thisx) (((thisx)->params >> 7) & 0xF)
```
Extra parenthesis aren't necessary, and just clearing up order of operations.